sys-process/parallel/parallel-20200422.ebuild

```diff
@@ -0,0 +1,31 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="A shell tool for executing jobs in parallel locally or on remote machines"
+HOMEPAGE="https://www.gnu.org/software/parallel/"
+SRC_URI="mirror://gnu/${PN}/${P}.tar.bz2"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
+
+RDEPEND="dev-lang/perl:=
+	dev-perl/Devel-Size
+	virtual/perl-Data-Dumper
+	virtual/perl-File-Temp
+	virtual/perl-IO"
+DEPEND="${RDEPEND}"
+
+src_configure() {
+	econf --docdir="${EPREFIX}"/usr/share/doc/${PF}/html
+}
+
+src_install() {
+	default
+
+	# See src/Makefile.am for this one:
+	rm -f "${ED}"/usr/bin/sem || die
```

Comment:
We dislike using `-f` when the action can be predicted. You should know when this file exists, and how. 

So just remove the `-f` option here. Rest looks good.

Reply:
Gone.